packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1022,6 +1033,9 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
             (this.mc.config.getBoolean(useDataStoreAliasingKey) ?? false) ||
             (runtimeOptions.useDataStoreAliasing ?? false);
 
+        this._chunkingDisabled = (this.mc.config.getBoolean(disableChunkingKey) ?? false);
+        this._maxOpSizeInBytes = (this.mc.config.getNumber(maxOpSizeInBytesKey) ?? defaultMaxOpSizeInBytes);
```

Comment:
BTW, I'd rather use kill-bit switches (i.e. enabled by default) vs. flighting.
And possibly just one key, with any value <= 0 meaning it's disabled.